Add text around client and origin ossification (#147)
ietf-wg-privacypass/base-drafts: draft-ietf-privacypass-auth-scheme.md

Origins MAY include multiple challenges, where each challenge refers to a different issuer
or a different token type, to allow clients to choose a preferred issuer or type.

An origin MUST NOT assume that token challenges will always yield a valid token.

> Clients may experience issues running the issuance protocol, e.g., because the attester

Reviewer: ```suggestion
Clients might experience issues running the issuance protocol, e.g., because the attester
```

> or issuer is unavailable, or clients may simply not support the requested token type.

Reviewer: ```suggestion
or issuer is unavailable, or clients might simply not support the requested token type.
```

Origins SHOULD account for such operational or interoperability failures by offering
clients an alternative type of challenge such as CAPTCHA for accessing a resource.

> To mitigate ossification around token dependencies, clients and servers SHOULD grease

Reviewer: ```suggestion
To mitigate the risk of deployments becoming dependent on tokens, clients and servers SHOULD grease
```

Reviewer 2: hi

> To mitigate the risk of deployments becoming dependent on tokens, clients and servers SHOULD grease

Reviewer 3: Should there be a recommendation for attesters to abort the issuance protocol with some non-zero probability?

Author: This is an interesting recommendation. For consistency reasons I could see it working, but it also might make tracking down issues that are isolated to either attester or issuer difficult in production. @tfpauly, thoughts?

Reviewer: I could see it working as an option. To some degree, it's out of scope for this auth scheme document, since we only know about origins and clients here. I'd probably leave it off for now?

Author: Yeah, I forgot this was in the auth scheme doc, which knows nothing about attesters or issuers.

> their behavior unless otherwise configured to do so. In particular, clients SHOULD

Reviewer: ```suggestion
their behavior unless explicitly configured not to. In particular, clients SHOULD
```
